helpers.js

```diff
@@ -31,6 +44,7 @@ async function fetchAllData(api) {
       debug(`fetching details for ${trip.name}...`);
       const tripDetailsRes = await getTripDetailsAsync(trip.id);
       const tripDetails = tripDetailsRes.body; // Again, assuming the body has the details in JSON format
+      downloadImage(tripDetails['closest_image_mobi_xlarge'],'./export/images/');
```

Comment:
@flyingbarron its probably best to add an await in front of the call to download image, just so all of this code runs in sequence. i can merge this in if you can make that change. no worries if not. appreciate the enhancement.